mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -966,11 +966,37 @@
     <string name="preference_follow_device_theme">Follow device theme</string>
 
     <!-- The title of the error page for websites that do not support HTTPS when HTTPS-Only is turned on  -->
-    <string name="errorpage_httpsonly_title">The site you requested does not support HTTPS</string>
-    <!-- The text of the error page for websites that do not support HTTPS when HTTPS-Only is turned on. %1$s will get replaced with the name of the app (e.g. "Firefox Focus") -->
-    <string name="errorpage_httpsonly_message">By default, %1$s attempts to connect using HTTPS for increased security. To change this setting or to learn more, go to Settings &gt; Privacy &amp; Security &gt; Security.</string>
+    <string name="errorpage_httpsonly_title" moz:RemovedIn="111" tools:ignore="UnusedResources">The site you requested does not support HTTPS</string>
+
+    <!-- The title of the error page for websites that do not support HTTPS when HTTPS-Only is turned on  -->
+    <string name="errorpage_httpsonly_title2">This site doesn’t support HTTPS</string>
 
-    <!-- Sessions List -->
+    <!-- The text of the error page for websites that do not support HTTPS when HTTPS-Only is turned on. %1$s will get replaced with the name of the app (e.g. "Firefox Focus") -->
+    <string name="errorpage_httpsonly_message" moz:RemovedIn="111" tools:ignore="UnusedResources">By default, %1$s attempts to connect using HTTPS for increased security. To change this setting or to learn more, go to Settings &gt; Privacy &amp; Security &gt; Security.</string>
+
+    <!-- The text of the error page for websites that do not support HTTPS when HTTPS-Only is turned on. %1$s will get replaced with the name of the app (e.g. "Firefox Focus").
+        Contains a button that will redirect the user to Privacy & Security Screen-->
+    <string name="errorpage_httpsonly_message2"><![CDATA[%1$s tries to use an HTTPS connection whenever possible for more security.
+     <a style="color:white;" href="%2$s">Learn more</a> <br/><br/>
+     Change this setting in Settings > Privacy &amp; Security > Security.]]></string>
```

Comment:
I don't think it's expected that the steps in this string are differently formatted as the ones in the string in `<string name="errorpage_httpsonly_message"`?
So:
* `go to Settings &gt; Privacy &amp; Security &gt; Security.`
versus here:
* `Change this setting in Settings > Privacy &amp; Security > Security.`